src/vm/CMakeLists.txt

```diff
@@ -202,7 +202,6 @@ set(VM_SOURCES_WKS
     gccover.cpp
     gcenv.ee.static.cpp
     gcenv.ee.common.cpp
-    gcenv.os.cpp
```

Comment:
This was an error - the VM would link in `gcenv.os.cpp` twice if building with a standalone GC and linking it into the runtime, which resulted in linker errors. Removing this source inclusion causes it to build once, as expected.